bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/MathUtils.java

```diff
@@ -63,4 +63,28 @@ public static long remainder(long numerator, long denominator) {
             }
         }
     }
+
+    public static long addExact(long num1, long num2) {
+        long r = num1 + num2;
```

Comment:
can't we say `result` instead of `r`? `r` appears not to have a meaning 🙂

Reply:
I will look into this.